docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md

```diff
@@ -1,48 +1,158 @@
 
-<Preface />
+In order to Validate data, we must first define a set of Expectations for that data to be validated against.  In this guide, we will walk you through processes of interactively creating and editing Expectations while simultaneously validating them against a Batch of data. By validating our Expectations as we define them we can quickly see if the Expectations we are setting are suitable for our data, and adjust them if necessary.
+
+:::tip Does this process edit my data?
+Not at all.  We call this process the "Interactive method" of creating and editing Expectations because validating the Expectations as we work requires interacting with a Batch of data.  This data is introspected to see if it fits the defined Expectation, but it is not edited: only examined.
+:::
+
+## Prerequisites
+
+<Prerequisites>
+
+- Great Expectations installed in a Python environment
+- Initialized a Filesystem Data Context to save your Expectations in
+- Created a Datasource from which to request a Batch of data for introspection
+- A passion for data quality
+
+</Prerequisites> 
+
+<details>
+<summary>
+
+### If you still need to set up Great Expectations...
+
+</summary>
+
+<IfYouStillNeedToSetupGX />
+
+</details>
+
+<details>
+<summary>
+
+### If you still need to initialize your Data Context...
+
+</summary>
+
+Please see the appropriate guide from the following:
+
+<DataContextInitializeQuickOrFilesystem />
+
+</details>
+
+<details>
+<summary>
+
+### If you still need to create a Datasource...
+
+</summary>
+
+Please reference the appropriate guide from the following:
+
+<ConnectingToDataFluently />
+
+</details>
 
 ## Steps
 
-### 1. Use the CLI to begin the interactive process of creating Expectations
-<UseTheCliToBeginTheInteractiveProcessOfCreatingExpectations />
+### 1. Import the Great Expectations module and instantiate a Data Context
+
+For this guide we will be working with Python code in a Jupyter Notebook. Jupyter is included with GX and provides a very convenient interface that lets us easily edit code and immediately see the result of our changes.
+
+The code to import Great Expectations and instantiate a Data Context is:
+
+```python name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py imports and data context"
+```
+
+:::info Data Contexts and persisting data
+
+If you are using an Ephemeral Data Context, your configurations will not persist beyond the current Python session.  If you are using a Filesystem or Cloud Data Context, they will.  The `get_context()` method will return the first of a Cloud or Filesystem Data Context if it can find one, but if it can't then it will provide you with an Ephemeral Data Context.  For more information on the `get_context()` method, please see our guide on [how to quickly instantiate a Data Context](/docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_quickly_instantiate_a_data_context).
```

Comment:
```suggestion
If you're using an Ephemeral Data Context, your configurations won't persist beyond the current Python session.  However, if you're using a Filesystem or Cloud Data Context, they do persist.  The `get_context()` method returns the first Cloud or Filesystem Data Context it can find. If a Cloud or Filesystem Data Context can't be located, it provides an Ephemeral Data Context. For more information about the `get_context()` method, see [How to quickly instantiate a Data Context](/docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_quickly_instantiate_a_data_context).
```